resources/linux/org.nickvision.tubeconverter.metainfo.xml.in

```diff
@@ -5,33 +5,52 @@
   <project_license>MIT</project_license>
```

Comment:
The project_license field here seems to be wrong

https://github.com/NickvisionApps/Parabolic?tab=GPL-3.0-1-ov-file

Reply:
Ah, let's fix that as well.